src/Session/Session.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Pantheon\Terminus\Session;
+
+use Terminus\Caches\FileCache;
+use Terminus\Models\User;
+
+class Session {
+  /**
+   * @var FileCache
+   */
+  protected $cache;
+
+  /**
+   * @var object
+   */
+  protected $data;
+
+  /**
+   * Instantiates object, sets session data
+   */
+  public function __construct($fileCache) {
+    $this->cache = $fileCache;
+    $session     = $this->cache->getData('session');
+    $this->data  = $session;
+    if (empty($session)) {
+      $this->data = new \stdClass();
```

Comment:
Is an anonymous object the best way to represent this? If it needs any methods, could it be a full fledged class like `Pantheon\Terminus\Session\Data` or would it be well served as an array?